backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/AddAccessRequest.java

```diff
@@ -1,7 +1,11 @@
 package com.redhat.cloud.notifications.routers.internal.models;
 
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
 import java.util.UUID;
 
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Comment:
`applicationId` as being received as such. Changed to receive it as `application_id`